hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -35,18 +36,48 @@ interface IMissingContainerResponse {
   pipelineID: string;
 }
 
+interface IContainerResponse {
+  containerID: number;
```

Comment:
```suggestion
  containerId: number;
```

Reply:
https://github.com/apache/ozone/blob/master/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/UnhealthyContainerMetadata.java

If we go through this Java File I can see  ContainerID and PipelineID

![image](https://user-images.githubusercontent.com/112169209/196702561-f7bcc102-ef94-4b1e-90d4-abf8821f9411.png)